account_invoice_change_currency/models/account_move.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2017-2018 Vauxoo
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    custom_rate = fields.Float(
+        digits=(12, 6),
+        default=1,
+        store=True,
+        readonly=False,
+        compute="_compute_currency_change_rate",
+        help="Set new currency rate to apply on the invoice.\n"
+        "This rate will be taken in order to convert amounts between the "
+        "currency on the invoice and last currency",
+    )
+    base_currency_id = fields.Many2one(
+        "res.currency",
+        help="Store the base currency when the invoice is created or the"
```

Comment:
Since field names and translations are being modified, you should update POT file to reflect those changes

Reply:
Done!